### PR TITLE
 [#256]调试器setting部分改版与优化

### LINKED ISSUE #256: 调试器功能和交互优化
优化界面交互，提升用户体验，提高重点功能使用效率。
主要需求内容：
．调试器页面UI样式及交互优化：突出扫码调试、在线更新。
．增加使用说明、常见问题、社区、开发文档入口及功能。
．增加FAQ功能。

### PR DESCRIPTION
**Pull request by charles-chai**
fix #256

### CODE REVIEW

**Review by yangxiangwu-oppo — APPROVED** [review]

**Review by yangxiangwu-oppo — APPROVED** [review]

**Review by duanchaodc — APPROVED** [review]

File: debug/shell/android/app-impl/src/main/java/org/hapjs/debugger/widget/CustomEditTextPreference.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package org.hapjs.debugger.widget;
```
Comment (jianghai33) [inline debug/shell/android/app-impl/src/main/java/org/hapjs/debugger/widget/CustomEditTextPreference.java:1]: copy right

File: debug/shell/android/app-impl/src/main/res/anim/bottom_in.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (jianghai33) [inline debug/shell/android/app-impl/src/main/res/anim/bottom_in.xml:1]: copy right

File: debug/shell/android/app-impl/src/main/java/org/hapjs/debugger/widget/CustomEditTextPreferenceDialogFragment.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
+package org.hapjs.debugger.widget;
```
Comment (jianghai33) [inline debug/shell/android/app-impl/src/main/java/org/hapjs/debugger/widget/CustomEditTextPreferenceDialogFragment.java]: copy right

**Review by jianghai33 — CHANGES_REQUESTED** [review]

**Review by duanchaodc — APPROVED** [review]

**Review by jianghai33 — APPROVED** [review]

**Review by duanchaodc — APPROVED** [review]

File: debug/shell/android/app-impl/src/main/java/org/hapjs/debugger/widget/CustomEditTextPreference.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2021, the hapjs-platform Project Contributors
```
Comment (chenyinli) [inline debug/shell/android/app-impl/src/main/java/org/hapjs/debugger/widget/CustomEditTextPreference.java:2]: copyright是2022

**Review by chenyinli — CHANGES_REQUESTED** [review]

**Review by duanchaodc — APPROVED** [review]

**Review by chenyinli — APPROVED** [review]

**Pull request merged**